rotkehlchen/db/upgrades/v24_v25.py

```diff
@@ -446,6 +446,85 @@ def upgrade_v24_to_v25(db: 'DBHandler') -> None:
     """
     helper = V24V25UpgradeHelper(db.msg_aggregator)
     cursor = db.conn.cursor()
+
+    # Create table misssing the creation statement in upgrades
+    cursor.execute("""
+    CREATE TABLE IF NOT EXISTS balancer_pools (
```

Comment:
Is this added at the same upgrade we introduced the table? It should be added when the table is supposed to exist for the first time, no?

Or you think it's fine to have it here since if it does not exist before, here is when it would break stuff?

Reply:
For what I saw in the affected code and the commit tags the tables were introduced in v1.15, the previous migration is from v1.14 and this is the next migration. I believe it is fine them being here since is the first step where it can fail and now it shouldn't

Reply:
All right. I trust you on this.